README.rst

```diff
@@ -105,6 +79,8 @@ Building the Driver on Linux and macOS
 
        $PHP_HOME/bin/php -dextension=modules/pdo_snowflake.so -m | grep pdo_snowflake
```

Comment:
Are there any additional instructions needed for this on CentOS?

This worked fine on Ubuntu, but on CentOS, the following error occurred:

```
[parallels@localhost pdo_snowflake]$ $PHP_HOME/bin/php -dextension=modules/pdo_snowflake.so -m | grep pdo_snowflake
PHP Warning:  PHP Startup: Unable to load dynamic library 'modules/pdo_snowflake.so' (tried: modules/pdo_snowflake.so (modules/pdo_snowflake.so: undefined symbol: uuid_unparse_lower), /usr/lib64/php/modules/modules/pdo_snowflake.so.so (/usr/lib64/php/modules/modules/pdo_snowflake.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
```

This was for the case where I downloaded and installed PHP.

If I build PHP instead and I use that built version, this error does not occur.

Reply:
I was build PHP from source code on Centos so didn't face such issue. I'll get a clean Centos to reproduce. At the same time, could you add following linelibsnowflakeclient/deps-build/linux/uuid/lib/libuuid.a \
after line 66 of scripts/build_pdo_snowflake.sh to see how it works?

Reply:
Thanks!

I added that line, ran the script again, and ran the command to load the driver.
The command succeeded without any errors.

Reply:
I've added this line in https://github.com/snowflakedb/pdo_snowflake/pull/231